.circleci/config.yml

```diff
@@ -37,8 +37,8 @@ commands:
                 apt install -y sudo
               fi
 
-              sudo apt update
-              sudo env DEBIAN_FRONTEND=noninteractive apt install -y curl git make unzip gcc xz-utils
+              sudo apt update --fix-missing
+              sudo env DEBIAN_FRONTEND=noninteractive apt install -y curl git make unzip gcc xz-utils python3-pip
```

Comment:
Hi @bartsmykla ,

The simplest way I could find to stay OS agnostic is using pip/pip3 from python.
I keep getting pip3 command not found or pip command not found.

Any idea what I'm missing here ?